nltk/corpus/__init__.py

```diff
@@ -206,6 +206,8 @@
     para_block_reader=tagged_treebank_para_block_reader, encoding='ascii')
 treebank_raw = LazyCorpusLoader(
     'treebank/raw', PlaintextCorpusReader, r'wsj_.*', encoding='ISO-8859-2')
+tweets = LazyCorpusLoader(
```

Comment:
I'd prefer this to be called `twitter_samples`; we've tended to use the same name for the corpus reader and the corpus directory.